### ISSUE TITLE
MPV broken in latest version

### ISSUE THREAD

**Issue by Yumae** [flathub/io.github.hydrusnetwork.hydrus#151]
Playback with MPV doesn't work at all. I get the following error when clicking on any animated file or video.
```
v573, linux, source
ValueError
('Invalid value for mpv parameter', -4, (<MpvHandle object at 0x7528f75ef5c0>, <mpv.LP_MpvNode object at 0x75291ec12b10>, <mpv.LP_MpvNode object at 0x75291ec12a80>))
  File "/app/hydrus/hydrus/client/gui/canvas/ClientGUICanvas.py", line 1544, in SetMedia
    Canvas.SetMedia( self, media )
  File "/app/hydrus/hydrus/client/gui/canvas/ClientGUICanvas.py", line 1232, in SetMedia
    self._media_container.SetMedia( self._current_media, maintain_zoom, maintain_pan )
  File "/app/hydrus/hydrus/client/gui/canvas/ClientGUICanvasMedia.py", line 2151, in SetMedia
    self._MakeMediaWindow()
  File "/app/hydrus/hydrus/client/gui/canvas/ClientGUICanvasMedia.py", line 1524, in _MakeMediaWindow
    self._media_window = CG.client_controller.gui.GetMPVWidget( self )
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/hydrus/hydrus/client/gui/ClientGUI.py", line 7366, in GetMPVWidget
    mpv_widget = ClientGUIMPV.MPVWidget( parent )
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/hydrus/hydrus/client/gui/canvas/ClientGUIMPV.py", line 293, in __init__
    self._player.loadfile( self._black_png_path )
  File "/app/lib/python3.11/site-packages/mpv.py", line 1329, in loadfile
    self.command('loadfile', filename.encode(fs_enc), mode, MPV._encode_options(options))
  File "/app/lib/python3.11/site-packages/mpv.py", line 1227, in command
    _mpv_command_node(self.handle, ppointer, out)
  File "/app/lib/python3.11/site-packages/mpv.py", line 142, in raise_for_ec
    raise ex
```

**Comment by bbhtt** [comment]
It is a upstream issue https://github.com/hydrusnetwork/hydrus/issues/1544

I'll downgrade MPV back to 0.37.0

**Comment by hydrusnetwork** [comment]
Hey, I just mentioned in my commit, this should be fixed on my end now. The new 1.0.6 'mpv' python package fixes the issue with 0.38.x libmpv. I don't know how flathub works, but if you rebuild the venv or whatever you need to do to reflect the new requirements.txt in hydrus, this should all be sorted.

**Comment by bbhtt** [comment]
Thank you, I updated python-mpv to 1.0.6
